### PR TITLE
DOCS-6350-SHA-256 comment removed

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

**Review by sebastiankorol — APPROVED** [review]

File: modules/ROOT/pages/sso-prerequisites-about.adoc [hunk 1]
```diff
@@ -35,5 +35,4 @@ https://anypoint.mulesoft.com/accounts/login/{your_org_domain}
 * OpenID Connect does not support role mapping.
 * If a SAML user belongs to certain groups, Anypoint Platform does not automatically grant equivalent roles in the organization.
```
Comment (kbostrom) [inline modules/ROOT/pages/sso-prerequisites-about.adoc:36]: Just a note: I reread this and I'm not sure that I understand how "certain groups" relates to "equivalent roles in the organization." If your audience does, then this is OK.

File: modules/ROOT/pages/sso-prerequisites-about.adoc [hunk 2]
```diff
@@ -35,5 +35,4 @@ https://anypoint.mulesoft.com/accounts/login/{your_org_domain}
...
 * Anypoint Platform does not generate the SAML assertion for single sign on. Your IdP generates the Sign On URL that you configure.
```
Comment (kbostrom) [inline modules/ROOT/pages/sso-prerequisites-about.adoc]:
```suggestion
* Anypoint Platform does not generate the SAML assertion for single sign on. 
Your IdP generates the sign-on URL that you configure.
```

File: modules/ROOT/pages/sso-prerequisites-about.adoc [hunk 3]
```diff
@@ -35,5 +35,4 @@ https://anypoint.mulesoft.com/accounts/login/{your_org_domain}
...
-* Only the SHA-1 algorithm is supported for SAML digital signatures. SHA-256 is not supported.
 * External Identity configuration is available only at the organization level. You cannot configure an external identity provider for a business group.
```
Comment (kbostrom) [inline modules/ROOT/pages/sso-prerequisites-about.adoc]:
```suggestion
* External identity configuration is available only at the organization level.
You cannot configure an external identity provider for a business group.
```

File: modules/ROOT/pages/sso-prerequisites-about.adoc (hunk 2 above)
Comment (kbostrom) [reply modules/ROOT/pages/sso-prerequisites-about.adoc]: GLOBALLY break steps or other list items after the first sentence or fragment, to increase scannability.

**Review by kbostrom — APPROVED** [review]
Here you go!

**Pull request merged**